Sources/PackageLoading/PackageExtensions.swift

```diff
@@ -145,6 +145,15 @@ extension Package {
             }
         }
 
+        /// Check for targets that are not mapped to any modules
+        let targetNames = Set(manifest.package.targets.map{ $0.name })
+        let moduleNames = Set(modules.map{ $0.name })
+        let diff = targetNames.subtracting(moduleNames)
```

Comment:
This is fine, but it would be cheaper to just compute the set of module names, and then look for any target name not in the set (no need to actually compute the subtraction).